### DISCUSSION TITLE
Loop out Stuck in Preimage Revealed for over 2 Weeks

### DISCUSSION THREAD

**Discussion by nostrbuild** [Ride-The-Lightning/RTL#1289] [Troubleshooting Questions] [answered]
I tried a loop out under services and it assigned a 1 sat/vb and so now it is stuck in the mempool, never to confirm.. I tried RBF, CPFP, nothing seeps to do anything..
The mempool now says two days, but it is really like 16 days:
https://mempool.space/tx/eeedb10423426e935b62b4dd4107b16e3ce2f53fa75ce5794434d32ecc073efe

Can I cancel this loop out?
Will it eventually time out?

Thank you
![Screenshot 2023-09-20 at 4 55 46 PM](https://github.com/Ride-The-Lightning/RTL/assets/122138811/89ed1ec2-e695-4796-a7ad-817481611650)

**Comment by saubyk** [comment, marked as answer]
Per my understanding, the next step after 'pre-image revealed' is settlement and sweeping of funds to your onchain wallet. Your node should typically bump it up automatically so that the funds are received in your wallet.

It might be worthwhile to open an issue in the Loop repo to get a better understanding of your options in this scenario.
https://github.com/lightninglabs/loop
